app/controllers/api/v1/bands_controller.rb

```diff
@@ -3,5 +3,9 @@ class Api::V1::BandsController < ApplicationController
   def index
     render json: Band.all
   end
+
+  def show
+    render json: Band.find(params[:id])
+  end
```

Comment:
im confused, this PR says its for the index page, but this is new code for...a show page?